docs/src/languages/php/tuning.md

```diff
@@ -56,12 +56,19 @@ PHP 7.4 and later supports preloading code files into shared memory once at serv
 bypassing the need to include or autoload them later.
 Depending on your application doing so can result in significant improvements
 to both CPU and memory usage.
-If using PHP 7.4 or later, see the [PHP Preload instructions](/languages/php/_index.md#opcache-preloading)
-for how to configure it on Platform.sh
-and consult your application's documentation to see
+If using PHP 7.4 or later, see the [PHP Preload instructions](./_index.md#opcache-preloading)
+for how to configure it on Platform.sh.
+Consult your application's documentation to see
 if they have any recommendations for an optimal preload configuration.
 
-If you are not using PHP 7.4, this is a good reason to upgrade.
+If you aren't using PHP 7.4, this is a good reason to upgrade.
+
+Note that the only way to clear the preload cache is by restarting PHP-FPM.
+PHP-FPM isn't restarted on every deployment automatically,
+so you might want to add that in a [`deploy` hook](../../configuration/app/hooks.md)
```

Comment:
Shouldn't way explain how? 
> so you might want to add `pkill -f php` to force restart PHP in a ...

Reply:
I believe it's `pkill -f php-fpm`, but the note could help, so I added it.

Reply:
Maybe `sv restart app` is a nicer way to do so?

Reply:
Changed it. :bow:

Reply:
ah the permission bug with `sv restart app` is gone? [It doesn't seem to be in Gitlab](https://github.com/orgs/platformsh/projects/3)

Reply:
It's in code review and it's a bug, so I think we can take its resolution as expected behavior.